tests/python/frontend/coreml/test_forward.py

```diff
@@ -422,6 +427,7 @@ def verify_unary_sqrt(input_dim):
 
 
 def verify_unary_rsqrt(input_dim, epsilon=0):
+    """Unary rsqrt"""
```

Comment:
Isn't it better to disable C0114 in pylint than to add doc-strings that are not useful?

Reply:
Actually I think so. But disabling C0114 in some functions is also inappropriate. So i just follow others.
@areusch any ideas?